server/src/test/java/org/elasticsearch/client/transport/TransportClientIT.java

```diff
@@ -67,7 +68,9 @@ public void testNodeVersionIsUpdated() throws IOException, NodeValidationExcepti
                 .put(Node.NODE_DATA_SETTING.getKey(), false)
                 .put("cluster.name", "foobar")
                 .put(TestZenDiscovery.USE_ZEN2.getKey(), true)
-                .put(ClusterBootstrapService.INITIAL_MASTER_NODE_COUNT_SETTING.getKey(), 1)
+                .putList(
+                    ClusterBootstrapService.INITIAL_MASTER_NODES_SETTING.getKey(), Collections.singletonList("testNodeVersionIsUpdated")
+                ).put(ClusterBootstrapService.INITIAL_MASTER_NODE_COUNT_SETTING.getKey(), 1)
```

Comment:
I think we only need to set one or other of these. Here I think it'd be better to set `INITIAL_MASTER_NODES_SETTING`.